openapi-cli/src/main/java/io/ballerina/openapi/generators/schema/TypeGeneratorUtils.java

```diff
@@ -179,6 +176,19 @@ public static void updateRecordFieldList(List<String> required,
         //Generate constraint annotation.
         AnnotationNode constraintNode = generateConstraintNode(fieldSchema);
         MetadataNode metadataNode;
+        boolean isConstraintSupport =
```

Comment:
Aren't we checking if this schema belongs to a union type? If so, shall we rename as follows?
```suggestion
        boolean isUnionType =
```